api/src/main/proto/stellarstation/api/v1/radio/radio.proto

```diff
@@ -161,6 +164,24 @@ message CCSDSTransferFrame {
   LineCoding line_coding = 6;
 }
 
+// https://gomspace.com/UserFiles/Subsystems/datasheet/gs-ds-nanocom-ax100-33.pdf
+//
+// ASM+Golay mode for GOM Space AX100 radio
+message ASMGolay {
```

Comment:
Why not name this `AX100`?

Reply:
I think the justification was that ASM + Golay was a particular framing mode of the AX100 radio. @vaelen can give more details.